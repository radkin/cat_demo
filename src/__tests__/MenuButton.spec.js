import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import MenuButton, { Counter, dataReducer } from '../Hamburger/MenuButton';

describe('MenuButton', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<MenuButton />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
