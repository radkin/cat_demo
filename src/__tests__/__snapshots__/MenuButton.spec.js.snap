// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuButton snapshot renders 1`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "alignItems": "center",
      "cursor": "pointer",
      "display": "flex",
      "flexDirection": "column",
      "height": "32px",
      "justifyContent": "center",
      "padding": "4px",
      "width": "32px",
    }
  }
>
  <div
    style={
      Object {
        "background": "black",
        "height": "2px",
        "marginBottom": "5px",
        "transform": "none",
        "transformOrigin": "top left",
        "transition": "all 0.2s ease",
        "width": "20px",
      }
    }
  />
  <div
    style={
      Object {
        "background": "black",
        "height": "2px",
        "opacity": 1,
        "transform": "none",
        "transition": "all 0.2s ease",
        "width": "20px",
      }
    }
  />
  <div
    style={
      Object {
        "background": "black",
        "height": "2px",
        "marginTop": "5px",
        "transform": "none",
        "transformOrigin": "top left",
        "transition": "all 0.2s ease",
        "width": "20px",
      }
    }
  />
</div>
`;
